src/themes/default/components/core/ProductTile.vue

```diff
@@ -85,9 +105,16 @@ export default {
     },
     visibilityChanged (isVisible, entry) {
       if (isVisible) {
-        if (config.products.configurableChildrenStockPrefetchDynamic && rootStore.products.filterUnavailableVariants) {
+        if (
+          config.products.configurableChildrenStockPrefetchDynamic &&
+          rootStore.products.filterUnavailableVariants
```

Comment:
Please don't use direct rootStore calls. Instead go for vuex methods e.g. this.dispatch for actions